cli/internal/runsummary/format_json.go

```diff
@@ -55,12 +55,13 @@ func (rsm *Meta) normalize() {
 // This struct exists solely for the purpose of serializing to JSON and should not be
 // used anywhere else.
 type nonMonorepoRunSummary struct {
-	ID                ksuid.KSUID        `json:"id"`
-	Version           string             `json:"version"`
-	TurboVersion      string             `json:"turboVersion"`
-	GlobalHashSummary *GlobalHashSummary `json:"globalCacheInputs"`
-	Packages          []string           `json:"-"`
-	EnvMode           util.EnvMode       `json:"envMode"`
-	ExecutionSummary  *executionSummary  `json:"execution,omitempty"`
-	Tasks             []*TaskSummary     `json:"tasks"`
+	ID                 ksuid.KSUID        `json:"id"`
+	Version            string             `json:"version"`
+	TurboVersion       string             `json:"turboVersion"`
+	GlobalHashSummary  *GlobalHashSummary `json:"globalCacheInputs"`
+	Packages           []string           `json:"-"`
+	EnvMode            util.EnvMode       `json:"envMode"`
+	FrameworkInference bool               `json:"frameworkInference"`
```

Comment:
Starting to think we're eventually just going to want something like:

```golang
	Config runOpts               `json:"config"`
```

Reply:
Yeah, I've got another one coming in `dotEnv`. I'm not excited about making a backward-incompatible change to the object shape right now, though. That should come with a minor at least.

Reply:
It could be backwards compatible by duplicating whatever else you're thinking of in two places. There are other things that fall into that boat. That said, I'm not even sure we want all of runOpts, or in the form they exist today. So this is not a change we need.